packages/rocketchat-theme/server/colors.less

```diff
@@ -155,8 +155,14 @@
 	background-color: @transparent-darker;
 }
 
-.background-transparent-darker-hover:hover {
-	background-color: @transparent-darker;
+@media all and(min-width: 781px) {
+	.background-transparent-darker-hover:hover {
```

Comment:
And to explain more: I'm not limiting the global classes `background-transparent-darker-hover` and `background-transparent-dark-hover`, just limiting their hover effects to desktop .